ui/client/components/QueriedStateTable.js

```diff
@@ -24,17 +25,19 @@ class QueriedStateTable extends React.Component {
   }
 
   render() {
+    const {t} = useTranslation()
```

Comment:
no.
hooks only in functional components.  https://reactjs.org/docs/hooks-overview.html
you need hoc (withTranslation) or component (Trans) here. https://react.i18next.com/latest/withtranslation-hoc
or rewrite to FC and use hook.

Reply:
thanks for catching, used hoc for now (this should be small change to merge into 0.4.0)